scripts/wsrep_sst_rsync.sh

```diff
@@ -62,15 +62,25 @@ check_pid_and_port()
     local rsync_pid=$2
     local rsync_port=$3
 
-    if ! which lsof > /dev/null; then
-      wsrep_log_error "lsof tool not found in PATH! Make sure you have it installed."
-      exit 2 # ENOENT
-    fi
+    case $OS in
+    FreeBSD)
+        local port_info=$(netstat -46lp ${rsync_port} 2>/dev/null | \
```

Comment:
@Sp1l  Is this a valid command on FreeBSD? I tried:
$ sudo netstat -46lp 4567
netstat: 4567: unknown or uninstrumented protocol

-p here expects 'protocol', but we are passing the rsync port.